.github/workflows/gh_pages.yml

```diff
@@ -27,7 +34,24 @@ jobs:
         cmake --build . --target doxygen-clang
         cmake --build . --target docs-sycl-html
         cmake --build . --target docs-clang-html
+        cd $GITHUB_WORKSPACE
+        mkdir rendered
+        cd rendered
+        yes | \cp -rf $GITHUB_WORKSPACE/build/tools/sycl/doc/html/* .
+        mv $GITHUB_WORKSPACE/build/tools/sycl/doc/doxygen/html doxygen/
+        mv $GITHUB_WORKSPACE/build/tools/clang/docs/html clang/
+        mv $GITHUB_WORKSPACE/build/tools/clang/docs/doxygen/html clang_doxygen/
+    - name: Pack
+      if: ${{ github.event_name != 'schedule' }}
+      run: tar -czf sycl_docs.tar.gz -C $GITHUB_WORKSPACE/rendered .
+    - name: Upload docs as artifact
+      if: ${{ github.event_name != 'schedule' }}
+      uses: actions/upload-artifact@v1
+      with:
+        name: sycl_docs
+        path: sycl_docs.tar.gz
```

Comment:
What is this for?

Reply:
It uploads build result as an artifact, so that we can check result before submitting this patch (or any other doc-changing patch, I think we can leave that part here).

Reply:
I don't think anyone is going to "check result before submitting this patch". How about we drop it for now and add as soon as we get the first request for that?

Reply:
Ok

Reply:
FYI, the size of sycl_docs.tar.gz is 2G, so uploading 2G for each pre-commit doesn't seem reasonable.
https://github.com/intel/llvm/pull/5160/checks?sha=cfab6d7f474b3b8d322afe4488be316bbac7aed5